cmd/jaeger/internal/integration/es_test.go

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) 2024 The Jaeger Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package integration
+
+import (
+	"testing"
+
+	"github.com/jaegertracing/jaeger/plugin/storage/integration"
+)
+
+func TestESStorage(t *testing.T) {
+	integration.SkipUnlessEnv(t, "elasticsearch")
+
+	s := &E2EStorageIntegration{
+		ConfigFile: "../../config-elasticsearch.yaml",
+		StorageIntegration: integration.StorageIntegration{
+			CleanUp:                      cleanUp,
```

Comment:
which `cleanUp` is this, where is it defined? It would be better to only refer to functions defined in the same file.

Reply:
it is defined in `badger_test.go`. 

its only purpose is to call Purge through HTTP.
should we move this into `e2e_integration.go`?

Reply:
yes, since it's shared. And I would rename it to `purge()`

Reply:
done